textdb/textdb-dataflow/src/main/java/edu/uci/ics/textdb/dataflow/queryrewriter/WordBase.java

```diff
@@ -0,0 +1,35 @@
+package edu.uci.ics.textdb.dataflow.queryrewriter;
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.net.URL;
+import java.util.HashSet;
+
+/**
+ * Created by shiladityasen on 4/30/16.
+ * Class to provide simple abstraction of a dictionary of words with constant retrieval time
+ * Used by FuzzyTokenizer to process query
+ */
+public class WordBase
```

Comment:
@sandeepreddy602 : I assume you will do the refactoring in this PR.  If so, @kishore-narendran , please wait for this refactoring to finish first.